programs/witness_node/main.cpp

```diff
@@ -298,6 +298,14 @@ void write_default_logging_config_to_stream(std::ostream& out)
    out << "# declare an appender named \"stderr\" that writes messages to the console\n"
           "[log.console_appender.stderr]\n"
           "stream=std_error\n\n"
+          "# declare an appender named \"default\" that writes messages to default.log\n"
+          "[log.file_appender.default]\n"
+          "# filename can be absolute or relative to this config file\n"
+          "filename=logs/default/default.log\n"
+          "# Rotate log every ? minutes, if leave out default to 60\n"
+          "rotation_interval=60\n"
+          "# how long will logs be kept (in days), if leave out default to 7\n"
```

Comment:
Default limit is 1 not 7 (please also change in p2p appender). https://github.com/bitshares/bitshares-core/blob/develop/programs/witness_node/main.cpp#L367